src/NorthwindWebsite/Filters/LoggingActionFilter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc.Filters;
+using NorthwindWebsite.Core.ApplicationSettings;
+
+namespace NorthwindWebsite.Filters;
+
+public class LoggingActionFilter : IActionFilter
+{
+    private readonly ILogger<LoggingActionFilter> _logger;
+    private readonly AppSettings _appSettings;
```

Comment:
Why do you need to store appsetting if you only need `_loggingEnabled`?

Reply:
Because I wat to get my settings concentrated and managed in one single place which isappsettings.json.
I can remove this settings from AppSettings class and get it with IConfiguration, but what's the practical difference as both are dependency injected. Only to make AppSettings ligher?

Reply:
No-no: my point of view was that `AppSetting` on class level variable here is redundant, it has no references, because you specify a boolean flag to control action logging on next row. Please, just remove this row, this is all I wanted :)